fMRI_Tutorial/DataCheck.ipynb

```diff
@@ -0,0 +1,473 @@
+      "text/plain": [
+       "<Figure size 1080x360 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Weight distribution plot\n",
+    "bins=np.arange(np.sqrt(len(np.concatenate(matdropped.values))))\n",
+    "bins = (bins - np.min(bins))/np.ptp(bins)\n",
+    "fig, axes = plt.subplots(1,2, figsize=(15,5))\n",
+    "\n",
+    "# Distribution of raw weights\n",
+    "rawdist = sns.distplot(matdropped.values.flatten(), bins=bins, kde=True, ax=axes[0])\n",
+    "rawdist.set(xlabel='Correlation Values', ylabel = 'Density Frequency')\n",
+    "\n",
+    "# Probability density of log10\n",
+    "log10dist = sns.distplot(np.log10(matdropped.values).flatten(), kde=True, ax=axes[1])\n",
+    "log10dist.set(xlabel='log(weights)')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### Now let's have a look at the DMN"
```

Comment:
Not completely clear to me why we would only plot the DMN? the idea would be that the DMN is more interconnected with itself than with the rest of the brain, and we can't really see that now. I see that you have that later on (after the normalization), but not clear to me what this really adds? 
Would it be an option to highlight/circle the DMN regions in the entire matrix? Think that is what you're trying to do at the end, but I get an error with df3. 
And perhaps average the (normalized!) correlation values of the PCUN within the DMN and outside and output them?

Reply:
I'm not sure about this part, maybe let's discuss it this Thursday? I've changed the jupyter quite a bit, have another try! I now put the color pallete in the beginning, so that it changes for the whole thing.

Reply:
I added new changes, now with the DMN things as well. Let me know what you think :)